lib/config.module.ts

```diff
@@ -19,6 +19,7 @@ import { ConfigFactoryKeyHost } from './utils';
 import { createConfigProvider } from './utils/create-config-factory.util';
 import { getRegistrationToken } from './utils/get-registration-token.util';
 import { mergeConfigObject } from './utils/merge-configs.util';
+import { globSync } from 'glob';
```

Comment:
Imports should be sorted by scoped->unscoped->relative, and alphabetically. (in VSCode, there's a command called "Organize Imports" - Shift+Alt+O)